cli/cmd/stat_test.go

```diff
@@ -20,8 +20,8 @@ func TestStat(t *testing.T) {
 
 		mockClient.StatSummaryResponseToReturn = &response
 
-		expectedOutput := `NAME    MESHED   SUCCESS      RPS   LATENCY_P50   LATENCY_P95   LATENCY_P99   SECURED
-emoji      1/2   100.00%   2.0rps         123ms         123ms         123ms      100%
+		expectedOutput := `NAME    MESHED   SUCCESS      RPS   LATENCY_P50   LATENCY_P95   LATENCY_P99   TLS
```

Comment:
i think you need one more space between `LATENCY_P99   TLS` to pass ci ;)

Reply:
noooooooo I thought fixed this!!! :(